thunder/tests/test_inplace_functionalization.py

```diff
@@ -551,6 +547,8 @@ def single_tensor_adam(
 
     jitted = executor.make_callable(single_tensor_adam)
     params, grads, exp_avgs, exp_avg_sqs = tensors
+    cd = thunder.compile_data(jitted)
+    cd.compile_options["torch_compile_fullgraph"] = False
```

Comment:
I am curious as to why this is required. I think a comment will be helpful.

Also, I think `jitted = executor.make_callable(single_tensor_adam, torch_compile_fullgraph=False)` would work. So, we won't need to grab the compile data and update it.

Reply:
Done.